map/map.js

```diff
@@ -433,37 +435,39 @@ var DefineMap = Construct.extend("DefineMap",{
 	// pass `key` to only log the matching property, e.g: `map.log("foo")`
 	log: function(key) {
 		//!steal-remove-start
-		var instance = this;
-
-		var quoteString = function quoteString(x) {
-			return typeof x === "string" ? JSON.stringify(x) : x;
-		};
+		if(process.env.NODE_ENV !== 'production') {
```

Comment:
same comment about log from above.  Btw, could `log` be shared?

Reply:
Even if they don't share the same event types?

Reply:
The log is shared now.